consensus/ibft/ibft.go

```diff
@@ -604,14 +605,14 @@ func (i *Ibft) buildBlock(snap *Snapshot, parent *types.Header) (*types.Block, e
 	}
 
 	// set the timestamp
-	parentTime := time.Unix(int64(parent.Timestamp), 0)
-	headerTime := parentTime.Add(defaultBlockPeriod)
+	parentTime := time.UnixMilli(int64(parent.Timestamp))
+	headerTime := parentTime.Add(time.Duration(i.blockTime) * time.Millisecond)
 
 	if headerTime.Before(time.Now()) {
 		headerTime = time.Now()
 	}
 
-	header.Timestamp = uint64(headerTime.Unix())
+	header.Timestamp = uint64(headerTime.UnixMilli())
```

Comment:
same as well

Reply:
You're linking the wrong commit but the change is there :) @ZeljkoBenovic  Resolving comment